assets/js/click-to-copy.js

```diff
@@ -1,86 +1,40 @@
-let codeListings = document.querySelectorAll('.highlight > pre');
+let codeListings = document.querySelectorAll(".highlight > pre");
```

Comment:
Consider running Prettier over this file, thanks: `npx prettier --single-quote --write assets/js/click-to-copy.js`

Reply:
Done